src/xrEngine/Device_Initialize.cpp

```diff
@@ -66,7 +66,8 @@ void CRenderDevice::Initialize()
 
         pcstr title = READ_IF_EXISTS(pSettingsOpenXRay, r_string, "window", 
                                     "title", "S.T.A.L.K.E.R.: Call of Pripyat");
-        
+
+        strncpy_s(Core.ApplicationTitle, title, sizeof(Core.ApplicationTitle) - 1);
```

Comment:
`xr_strcpy` is save and it's enough, hence there's no need to use `strncpy_s`.
```suggestion
        xr_strcpy(Core.ApplicationTitle, title);
```

Reply:
Indeed. Thank you